### PR TITLE
first draft

### PR DESCRIPTION
**Pull request by mleager**

### CODE REVIEW

File: modules/rds/sm.tf
Comment (mleager) [inline modules/rds/sm.tf:1]: deleted this file

File: modules/asg/security_group.tf [hunk 1]
```diff
@@ -11,6 +11,14 @@ resource "aws_security_group" "private" {
     security_groups = [var.load_balancer_sg]
   }
 
+  ingress {
```
Comment (geoffrey1330) [inline modules/asg/security_group.tf:14]: you don't have to set this ingress rule

Comment (geoffrey1330) [reply modules/asg/security_group.tf:14]: export asg security group id into rds

File: modules/rds/db.tf [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+resource "aws_db_instance" "mysql" {
+  #availability_zone       =
+  allocated_storage       = 10
+  backup_retention_period = 7
+  db_name                 = "mydb"
+  engine                  = "mysql"
+  engine_version          = "8.0.33"
+  instance_class          = "db.t3.micro"
+  username                = "mark"
+  password                = var.password
```
Comment (geoffrey1330) [inline modules/rds/db.tf:18]: you don't have any variable called password

**Review by geoffrey1330 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**